boomer/src/main.rs

```diff
@@ -205,6 +205,8 @@ fn main() -> Result<()> {
         .output_from_connector(&connector)
         .context("Couldn't find a valid output for that connector")?;
 
+    info!("Using output: {}", output);
+
```

Comment:
Once those changes land, we should probably update the cargo file to use a revision instead of just a git tree. It will make reproducibility easier.